Addition of GPU_CORE_FREQUENCY_STEP for NVIDIA GPUs (#2721)
geopm/geopm: service/docs/source/geopm_pio_nvml.7.rst

    *  **Format**: double
    *  **Unit**: hertz

``NVML::GPU_CORE_FREQUENCY_STEP``

>     The Streaming Multiprocessor frequency step size in hertz.  If the step size is variable the average of all steps is provided.

Reviewer: ```suggestion
    The Streaming Multiprocessor frequency step size in hertz.  The average step size is provided in the case where the step size is variable.
```
